drivers/storage/portworx/portworx.go

```diff
@@ -990,8 +990,10 @@ func (p *portworx) GetKVDBMembers(cluster *corev1.StorageCluster) (map[string]bo
 		return nil, fmt.Errorf("error in connecting to portworx sdk server: %s", err)
 	}
 	serviceClient := pxapi.NewPortworxServiceClient(p.sdkConn)
-	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
-	defer cancel()
+	ctx, err := pxutil.SetupContextWithToken(context.Background(), cluster, p.k8sClient)
```

Comment:
Why are we removing the timeout context? We can still create that and pass it to SetupContextWithToken instead of passing a Background context.

Reply:
Passing the context to create token only if security is enabled.